### ISSUE TITLE
Unable to display interface details 

### ISSUE THREAD

**Issue by BooBox26** [librenms/librenms#16129]
### The problem

Hello,

Since the last update Sun Jun 16 2024, we can no longer display the details of an interface, I tested on 3 different LibreNMS (including the demo version) I get the same error message.

Here's what I got in the logs:
View [device.tabs.includes.port_row] not found. {"userId":306,"exception":"[object] (InvalidArgumentException(code: 0): View [device.tabs.includes.port_row] not found. at /opt/librenms/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:137)"}

Can you see what's wrong with it?

Thanks you !

Best regards,
Nathan
![librenms-crash](https://github.com/librenms/librenms/assets/79327440/d4cac7e1-954c-4a57-8c05-b3acce833cbe)

### Output of ./validate.php

```txt
===========================================
Component | Version
--------- | -------
LibreNMS  | 24.6.0-1-g075ba4c93 (2024-06-16T18:29:06+02:00)
DB Schema | 2024_04_29_183605_custom_maps_drop_background_suffix_and_background_version (294)
PHP       | 8.2.7
Python    | 3.11.2
Database  | MariaDB 10.11.4-MariaDB-1~deb12u1
RRDTool   | 1.7.2
SNMP      | 5.9.3
===========================================

[OK]    Composer Version: 2.7.7
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database Schema is current
[OK]    SQL Server meets minimum requirements
[OK]    lower_case_table_names is enabled
[OK]    MySQL engine is optimal
[OK]    Database and column collations are correct
[OK]    Database schema correct
[OK]    MySQL and PHP time match
[OK]    Active pollers found
[OK]    Dispatcher Service not detected
[OK]    Locks are functional
[OK]    Python poller wrapper is polling
[OK]    Redis is unavailable
[OK]    rrdtool version ok
[OK]    Connected to rrdcached
```

### What was the last working version of LibreNMS?

24.5.0

### Anything in the logs that might be useful for us?

```txt
cat /opt/librenms/logs/librenms.log

View [device.tabs.includes.port_row] not found. {"userId":306,"exception":"[object] (InvalidArgumentException(code: 0): View [device.tabs.includes.port_row] not found. at /opt/librenms/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:137)"}
```

**Comment by grainconnect-imord** [comment]
I am also seeing this. And all of our VLAN interfaces no longer display on device -> ports.

**Comment by Jarod2801** [comment]
We have the same issue.

Also in the list deactivated ports are not shown anymore.

**Comment by Jarod2801** [comment]
`php artisan optimize:clear` fixed it for me

**Comment by krvi** [comment]
> `php artisan optimize:clear` fixed it for me
> 
> Read it on the discord channel. @murrant mentioned it there.

That did not solve it for me. Still getting the same error:
```
View [device.tabs.includes.port_row] not found. {"userId":4,"exception":"[object] (InvalidArgumentException(code: 0): View [device.tabs.includes.port_row] not found. at /opt/librenms/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:137)"}
```

**Comment by murrant** [comment]
fixed by #16124
